src/bytecode/astgen.rs

```diff
@@ -1410,7 +1437,7 @@ mod tests {
         let fct = code("fun f() -> Int { let x = 1; return x; }");
         let expected = vec![ConstInt(r(0), 1), RetInt(r(0))];
         assert_eq!(expected, fct.code());
-    }
+    }    
```

Comment:
I think you added these spaces by mistake

Reply:
Yep, I will remove them.